app/models.py

```diff
@@ -61,12 +62,14 @@ class PPE(db.Model):
     sku = db.Column(db.String(16), index=True)
     desc = db.Column(db.String(200))
     img = db.Column(db.BLOB())
-    wants = db.relationship('Wants', backref='author4', lazy='dynamic')
-    has = db.relationship('Has', backref='author5', lazy='dynamic')
-    exchange = db.relationship('Exchange', backref='author6', lazy='dynamic')
+    wants = db.relationship('Wants', backref='ppe')
+    has = db.relationship('Has', backref='ppe')
+    # FIXME: there's already a `ppe` property on Exchange (should be called ppe_id), but I don't
+    # want to change it.
+    exchange = db.relationship('Exchange', backref='ppe_ref')
 
     def __repr__(self):
-        return '<PPE {}>'.format(self.sku)
+        return '<PPE id={} sku={}>'.format(self.id, self.sku)
```

Comment:
I changed `__repr__` on most (all?) of the models to dump out basically all of the information for easier debugging.